### PR TITLE
hotfix to correct times got from db, also doumentation changes

### PR DESCRIPTION
**Pull request by TBTerra**
changes the time that spawns are checked when is -ss mode and are pulling their data from db

## Description
the original commit had the database return the despawn time rather than the (est) spawn time, causing it to not work when -ss was used without a fill, and also causing --dump-spawnpoints to generate an incorrect file

also ive updated the -ss documentation to not have so many formatting/spelling errors

## Motivation and Context
Motivation: fix it before people start blaming me

## How Has This Been Tested?
only tests over the original is that --dump-spawnpoints was used to get the output in human readable format to check that spawn times where not off by 15mins

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: pogom/search.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def search_worker_thread_ss(args, account, search_items_queue, parse_lock, encry
                                 log.exception('Search step %s map parsing failed, retrying request in %g seconds. Username: %s', step, sleep_time, account['username'])
                                 failed_total += 1
                         time.sleep(sleep_time)
-                    time.sleep(sleep_time)
+                    time.sleep(args.scan_delay)
```
Comment (invisiblek) [inline pogom/search.py]: is this relevant to this change? seems out of place

Comment (TBTerra) [inline pogom/search.py]: its not technicaly part of the change noted, i can remove it if you want

Comment (invisiblek) [inline pogom/search.py]: i guess i'd prefer it in a separate commit since its unrelated

Comment (TBTerra) [inline pogom/search.py]: reverted change
